src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsRecoveryObject.java

```diff
@@ -85,4 +85,9 @@ public void verifyPageFairRecoveryWithNoAdBlock() {
     // List<WebElement> markedSlots = driver.findElements(By.cssSelector("[adonis-marker]"));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)